io/net/bonding.py

```diff
@@ -128,12 +128,11 @@ def setup_ip(self):
         output = self.session.cmd(cmd)
         result = ""
         result = result.join(output.stdout.decode("utf-8"))
-        self.peer_first_interface = result.split()[0]
+        self.peer_first_interface = result.split()[-1]
```

Comment:
can you please explain ?

Reply:
result.split()[0] here not getting interface name, its coming inet.